floris/tools/optimization/yaw_optimization/yaw_optimization_base.py

```diff
@@ -255,16 +236,14 @@ def _reduce_control_problem(self):
         user-specified set of bounds (where bounds[i][0] == bounds[i][1]),
         or alternatively turbines that are far downstream in the wind farm
         and of which the wake does not impinge other turbines, if
-        exclude_downstream_turbines == True. This function also reduces
-        the optimization problem by exploiting layout symmetry, if
-        exploit_layout_symmetry == True.
+        exclude_downstream_turbines == True.
         """
         # Initialize which turbines to optimize for
         self.turbs_to_opt = (self.maximum_yaw_angle - self.minimum_yaw_angle >= 0.001)
 
         # Initialize subset variables as full set
         self.fi_subset = self.fi.copy()
-        nwinddirections_subset = copy.deepcopy(self.fi.floris.flow_field.n_wind_directions)
+        n_wind_directions_subset = copy.deepcopy(self.fi.floris.flow_field.n_findex)
```

Comment:
Should this be `n_findex_subset` for consistency?

Reply:
Good call.